modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/ServiceDetailsViewManager.java

```diff
@@ -220,11 +220,42 @@ public List<DeployedService> listDeployedServicesOfIsv(
             String serviceName,
             String serviceVersion,
             ServiceDeploymentState state) {
+        String serviceVendor = userServiceHelper.getCurrentUserManageIsv();
+        return listDeployedServicesOfRole(
+                category, csp, serviceName, serviceVersion, state, serviceVendor);
+    }
 
+    /**
+     * Use query model to list CSP deployment services.
+     *
+     * @param category of the services to be filtered.
+     * @param csp of the services to be filtered.
+     * @param serviceName of the services to be filtered.
+     * @param serviceVersion of the services to be filtered.
+     * @param state of the services to be filtered.
+     * @return serviceVos
+     */
+    public List<DeployedService> listDeployedServicesOfCsp(
```

Comment:
we must remove the word list from all methods. 
better would be - getAllDeployedServicesByCsp